src/ApiApprover/ApiApprover.csproj

```diff
@@ -1,13 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net40</TargetFramework>
```

Comment:
As long as you're keeping this package around, I'd suggest multi-targeting it as `<TargetFrameworks>netstandard2.0;net40</TargetFrameworks>` as well.

Reply:
Like mentioned in the description I can't multi-target because ApprovalTests only supports netstandard2.0

Reply:
Ah, I didn't notice that. Then ApprovalTests is doing the wrong thing.

At this point, you should always be multi-targeting to at least `net461;netstandard2.0`. Just targeting `netstandard2.0` is basically the wrong thing to do because support for .NET Standard 2.0 is broken in various ways in `net461` - `net471`.